File: waterproof/src/main/java/mx/yellowme/waterproof/mvp/simple/SimpleView.java
package mx.yellowme.waterproof.mvp.simple;

import mx.yellowme.waterproof.mvp.BaseView;

/**
 * Created by migdonio on 7/28/17.
 */

public interface SimpleView<Element> extends BaseView {
    void showElement(Element element);
    void setEmptyState(boolean isEmpty);
}

File: waterproof/src/main/java/mx/yellowme/waterproof/mvp/simple/SimpleView.kt
package mx.yellowme.waterproof.mvp.simple

import mx.yellowme.waterproof.mvp.BaseView

/**
 * Created by migdonio on 7/28/17.
 */

interface SimpleView<in Element> : BaseView {
    fun showElement(element: Element)
    fun setEmptyState(isEmpty: Boolean)
}
